index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="about-me">
+            <div id="ab-me-img"></div>
+            <div id="ab-me-1-img"></div>
+            <div id="ab-me-desktop-img"></div>
+            <div id="ab-me-desktop-1-img"></div>
+
+            <h1 class="title">About me</h1>
+            <p class="text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <button class="see-btn">
+                <p class=".text">
+                    Get my resume
+                </p>
+            </button>
+        </section>
+
+        <section id="skills">
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/languages.svg" />
+                </div>
```

Comment:
- [ ] - [ ] [OPTIONAL]  I would suggest you to remove `div` tags from around  `img` tags from your portfolio .  Unless you need to include the `div` for some structural/organisational reason you're likely better off just styling the element tag itself or you can just use class name or id names for styling that component. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.